caampr-backend/src/main/java/com/jelistan/caampr/lambda/dao/GearDao.java

```diff
@@ -0,0 +1,85 @@
+package com.jelistan.caampr.lambda.dao;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBQueryExpression;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.amazonaws.services.dynamodbv2.model.ComparisonOperator;
+import com.amazonaws.services.dynamodbv2.model.Condition;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.jelistan.caampr.lambda.model.Gear;
+import com.jelistan.caampr.lambda.model.GearListRequest;
+import com.jelistan.caampr.lambda.model.VisibilityTypes;
+
+import javax.inject.Inject;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Data Access Object for the Gear table
+ */
+public class GearDao {
```

Comment:
Do git diffs not have a better way of showing refactoring of files than to say that the old one (dynamogetter or whatever it was) was deleted, and the new one was created from scratch?

Reply:
not really.  It's typically best practice to do the "moves/renames" in a single commit so the reviewer is only looking at that crap and not real code changes.  Then after that is pushed, do real code changes.